stablehlo/dialect/Serialization.cpp

```diff
@@ -15,20 +15,27 @@ limitations under the License.
 
```

Comment:
Since this is a backport from MLIR-HLO, let's note that in the commit message. I've also tagged the PR accordingly.